packages/okta-react/test/jest/auth.test.js

```diff
@@ -0,0 +1,11 @@
+import Auth from '../../src/Auth';
+
+const pkg = require('../../package.json');
+
+test('Auth component sets the right user agent on AuthJS', () => {
+  const auth = new Auth({
+    issuer: 'https://foo/oauth2/default'
+  });
+  const expectedUserAgent = `${pkg.name}/${pkg.version} okta-auth-js-`;
```

Comment:
Shouldn't this pickup the `okta-auth-js` version too?

Reply:
To assert that, you need to know which version is currently depended on.  Ideas I had that I didn't love:
* Lock package.json down to a specific patch version
* Fish into node modules to see what's currently installed.
* Use a regex in the matcher, to at least assert the whole thing looks like what we expect